controller/rest/rest.go

```diff
@@ -1759,6 +1759,12 @@ func StartRESTServer() {
 	r.DELETE("/v1/api_key/:name", handlerApikeyDelete)
 	r.GET("/v1/selfapikey", handlerSelfApikeyShow) // Skip API document
 
+	// remote export repository
+	r.POST("/v1/remote_export_repository", handlerRemoteExportRepositoryPost)
+	r.GET("/v1/remote_export_repository/:alias", handlerRemoteExportRepositoryGet)
```

Comment:
This will require 2 API requests for loading system configuration page.
Why not follow the similar path as webhook configurations?

Reply:
I will implement a functionality to retrieve all configured remote repositories.

As for best practices for the `system/config` store, I'll message you.